internal/app/api/v1/testtriggers.go

```diff
@@ -116,10 +116,9 @@ func (s *TestkubeAPI) UpdateTestTriggerHandler() fiber.Handler {
 		testTrigger.Spec = crdTestTrigger.Spec
 		testTrigger.Labels = request.Labels
 		testTrigger.Annotations = request.Annotations
-		testTrigger, err = s.TestTriggersClient.Update(testTrigger)
 
 		s.Metrics.IncUpdateTestTrigger(err)
-
+		testTrigger, err = s.TestTriggersClient.Update(testTrigger)
```

Comment:
Should be back the other way around. Otherwise `err` won't be passed to the metric